beacon-chain/core/state/state_transition.go

```diff
@@ -175,3 +194,11 @@ func validatorSetRecalculations(
 
 	return append(shardAndCommittesForSlots[params.BeaconConfig().CycleLength:], newShardCommitteeArray...), nil
 }
+
+// createRandaoMix sets the block randao seed into a beacon state randao.
```

Comment:
Can you clarify this? What is it setting exactly? Is there anything you can link to wrt why this is the way it is, in the comment?

Reply:
It XORs block randao seed with the state randao seed. Maybe we should update the comment for better clarity